repository/kafka/operator/templates/statefulset.yaml

```diff
@@ -23,7 +23,7 @@ spec:
       containers:
         - name: k8skafka
           imagePullPolicy: Always
-          image: mesosphere/kafka:0.1-2.2.1
+          image: mesosphere/kafka:0.1.0-2.2.1
```

Comment:
Is the version here supposed to be independent from the operator version?

Reply:
yes, `0.1.0` isn't related to operator version